frontend/src/main/scala/bloop/data/Project.scala

```diff
@@ -153,6 +166,19 @@ final case class Project(
     cp.toArray
   }
 
+  def rawRuntimeClasspath: List[AbsolutePath] = this.platform match {
```

Comment:
Should we inline this method and `pickValidRuntimeResources` in `fullRuntimeClasspath`? They don't have other callers and it's better if we just encapsulate them under `fullRuntimeClasspath`.